src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1495,6 +1495,22 @@ private function registerMessengerConfiguration(array $config, ContainerBuilder
             throw new LogicException(sprintf('The default bus named "%s" is not defined. Define it or change the default bus name.', $config['default_bus']));
         }
 
+        $senderAliases = array();
+        foreach ($config['transports'] as $name => $transport) {
+            if (0 === strpos($transport['dsn'], 'amqp://') && !$container->hasDefinition('messenger.transport.amqp.factory')) {
+                throw new LogicException('The default AMQP transport is not available. Make sure you have installed and enabled the Serializer component. Try enable it or install it by running "composer require symfony/serializer-pack".');
```

Comment:
I know exception messages must end with a dot, but I'm afraid some people will execute `composer require symfony/serializer-pack .` I don't know if this happens in other error messages or if we reword them to never show a console command at the end.

Reply:
It's wrapped in `"` so I believe it's clear that the dot is not part of the command :)

Reply:
Agree with `@sroze`, copy-pasting with quotes is broken and the composer exception thrown when adding the dot is clear enough about the mistake. We have a bunch of similar messages in ControllerTrait